integration/util/waysnodes/nodes2wayblotdb/db.go

```diff
@@ -0,0 +1,214 @@
+// Package nodes2wayblotdb stores `fromNodeID,toNodeID -> wayID` mapping in blotdb.
+package nodes2wayblotdb
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+// DB stores `fromNodeID,toNodeID -> wayID` in blotdb.
+type DB struct {
+	db *bolt.DB
+}
+
+const (
+	defaultBucket = "bucket"
+)
+
+var (
+	errEmptyDB     = errors.New("empty db")
+	errKeyNotFound = errors.New("key not found")
```

Comment:
Here only defines common errors instead of all errors: will be used more than once. I saw many packages define some exported errors at the beginning, I think that's good since callers can handle them easer.